src/main/java/nextstep/subway/path/ui/PathController.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.subway.path.ui;
+
+import lombok.RequiredArgsConstructor;
+import nextstep.subway.path.application.PathService;
+import nextstep.subway.path.dto.PathResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/paths")
+public class PathController {
+
+    private final PathService pathService;
+
+    @GetMapping
+    public ResponseEntity findShortestPath(@RequestParam Long source, @RequestParam Long target) {
```

Comment:
```suggestion
    public ResponseEntity<PathResponse> findShortestPath(@RequestParam Long source, @RequestParam Long target) {
```

제네릭에 타입을 명시해주는게 좋겠네요

Reply:
제네릭 사용! 👍 
피드백 감사드립니다 :)